src/blocks/battery.rs

```diff
@@ -232,7 +237,10 @@ impl UpowerDevice {
             .block_error("battery", "Failed to read UPower Type property.")?;
 
         // https://upower.freedesktop.org/docs/Device.html#Device:Type
-        if upower_type != 2 {
+        // consider any peripheral, UPS and internal battery
+        let list: [u32; 5] = [2,3,4,5,6];
+
+        if !list.contains(&upower_type) {
```

Comment:
It may be better to simply disallow type `1` ("Line Power"). I'd also appreciate if the link to the docs remained in a comment so we have it for future reference (as happened here).

Reply:
Done